manim/utils/config_utils.py

```diff
@@ -130,23 +133,46 @@ def _parse_cli(arg_list, input=True):
         description="Animation engine for explanatory math videos",
         epilog="Made with <3 by the manim community devs",
     )
+    min_argvs = 4 if "py" in sys.argv[0] else 2
+    if input and (len(sys.argv) > min_argvs-1 # If "manim" is not the only command
+        and any(a == item for a in sys.argv for item in NON_ANIM_UTILS) #non-anim exists
+        or len(sys.argv) == min_argvs-1):
+
+        subparsers = parser.add_subparsers(dest="subcommands")
+        cfg_related = subparsers.add_parser('cfg')
+        cfg_subparsers = cfg_related.add_subparsers(dest="cfg_subcommand")
+
+        cfg_write = cfg_subparsers.add_parser('write')
+        cfg_write.add_argument(
+            "--level",
+            choices=["user", "cwd"],
+            default=None,
+            help="Specify if this config is for user or just the working directory."
+            )
+        cfg_show = cfg_subparsers.add_parser('show')
+
+        cfg_export = cfg_subparsers.add_parser("export")
+        cfg_export.add_argument("--dir",default=os.getcwd())
```

Comment:
Comment: the variable names are confusing me... these are ArgParser objects, correct? I'd suggest something like `cfg_show_parser`.

Reply:
Actually, do we even need to use `cfg_show` anywhere else in the code? If not, don't store it in a variable.

Reply:
>  I'd suggest something like cfg_show_parser.

I've changed the names as suggested, and also stopped storing `cfg_show` as a variable in the latest commit. Thanks for pointing them out!